editors/mediawiki.el/Portfile

```diff
@@ -1,37 +1,39 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 PortSystem 1.0
+PortGroup github 1.0
 
-name                    mediawiki.el
-version                 2.2.4.1
+github.setup            hexmode mediawiki-el 2.3.1
+set myname              mediawiki.el
```

Comment:
this should be `name mediawiki.el` as also pointed out by the lint error

Reply:
ok

Reply:
ok fixed in c15660c